client/src/main/java/org/spine3/protobuf/EntityPackagesMap.java

```diff
@@ -62,7 +62,7 @@
      * <p>Example:
      * <p>{@code com.example.order} - {@code AGGREGATE}
```

Comment:
But we have a convention that there is a separate package for aggregate's files. Each package contains files with aggregate state, `commands.proto` (which must have exactly this name) and `events.proto`.